<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommPara</class>
 <widget class="QWidget" name="CommPara">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>680</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>69</y>
     <width>1001</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>IO输出</string>
   </property>
   <widget class="QLineEdit" name="OKIOOutputProtEdit">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>72</y>
      <width>110</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="NGIOOutputProtEdit">
    <property name="geometry">
     <rect>
      <x>396</x>
      <y>72</y>
      <width>110</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>78</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>OK IO输出：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>295</x>
      <y>78</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>NG IO输出：</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="AdvancedButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>高级</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="BasisButton">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>40</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>基础</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxTrigger">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>197</y>
     <width>1001</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>IO触发</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QComboBox" name="IOTriggerWayCbB">
    <property name="geometry">
     <rect>
      <x>396</x>
      <y>43</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="IOTriggerPortCbB">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>44</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>46</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>I触发端口：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>294</x>
      <y>47</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>IO触发方式：</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>378</y>
     <width>1001</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>标定参数选择</string>
   </property>
   <widget class="QLineEdit" name="lineEditScaleParamName">
    <property name="geometry">
     <rect>
      <x>396</x>
      <y>46</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParamName">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>46</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>47</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>多点标定参数：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>279</x>
      <y>49</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>刻度校准参数：</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxCmd">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>296</y>
     <width>1001</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>命令触发</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QLineEdit" name="lineEdit_Cmd_trigger_string">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>30</y>
      <width>201</width>
      <height>35</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>37</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>命令字：</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>6</y>
     <width>1001</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>任务重命名</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>29</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>工具别名：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtModifyTaskName">
    <property name="geometry">
     <rect>
      <x>137</x>
      <y>25</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnModifyTaskName">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>24</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>修改</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStep">
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>30</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnResetTaskName">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>24</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInfor">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>30</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>1001</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Cmos参数选择</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>31</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cmos参数：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Coms_Param">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>30</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
